kafka/src/main/scala/hydra/kafka/producer/KafkaRecordMetadata.scala

```diff
@@ -1,15 +1,25 @@
 package hydra.kafka.producer
 
-import hydra.core.transport.RecordMetadata
+import hydra.core.transport.{AckStrategy, RecordMetadata}
+import spray.json.{DefaultJsonProtocol, JsNumber, JsObject, JsString, JsValue, RootJsonFormat}
 
 /**
   * Created by alexsilva on 2/22/17.
   */
-case class KafkaRecordMetadata(offset: Long, timestamp: Long, topic: String, partition: Int,
-                               deliveryId: Long) extends RecordMetadata
+case class KafkaRecordMetadata(offset: Long, timestamp: Long, destination: String, partition: Int,
```

Comment:
This is still a KafkaRecord, doesn't it make sense to keep the field name as "topic"? I understand wanting to be decoupled from kafka but this case class isn't